pkgs/development/python-modules/ovito/default.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchurl,
+  autoPatchelfHook,
+  numpy,
+  traits,
+  pyside6,
+}:
+
+buildPythonPackage {
+  pname = "ovito";
+  version = "3.14.0";
+  pyproject = true;
```

turns out, `format = "wheel";` fitted more (please, don't blindly accept suggestions, test them yourself)
```suggestion
  format = "wheel";
```